webapp/portlet/src/main/webapp/WEB-INF/gatein-resources.xml

```diff
@@ -302,6 +302,21 @@
     <skin-name>Enterprise</skin-name>
     <css-path>/skin/css/standalone-pages/AccountSetup.css</css-path>
   </portlet-skin>
+
+  <portlet-skin>
+    <application-name>social-portlet</application-name>
+    <portlet-name>VerticalMenu</portlet-name>
+    <skin-name>Enterprise</skin-name>
+    <css-path>/skin/css/portlet/VerticalMenu/Style.css</css-path>
+    <css-priority>1</css-priority>
+  </portlet-skin>
+
+  <portal-skin>
```

Comment:
```suggestion
  <portlet-skin>
```

Can you manage to add this in `VerticalMenu` portlet skin ? Is it reused elsewhere ?

Reply:
it is used also in the hamburger menu

Reply:
> it is used also in the hamburger menu

ok, you can import the less in both Modules instead of adding a new HTTP call. (unifying by less instead of unifying by addin anew HTTP call in browser side)

Reply:
done